packages/core/src/lib/context-menu/floating-menu.svelte

```diff
@@ -5,21 +5,24 @@ import { clickOutside } from '$lib/click-outside';
 import { floatingStyle, type FloatingMenuPlacement } from './floating-style';
 import ContextMenu from './context-menu.svelte';
 
+export let isOpen: boolean;
+export let label: string | undefined = undefined;
+export let describedBy: string | undefined = undefined;
 export let placement: FloatingMenuPlacement = 'bottom-start';
 export let offset = 0;
 export let buttonCX: cx.Argument = '';
 export let menuCX: cx.Argument = '';
+export let onChange: (isOpen: boolean) => unknown;
```

Comment:
[q] what is the thought behind this versus dispatching an event like the modal?

Reply:
It seems like Svelte is moving away from `on:...` handlers and [towards function props](https://youtu.be/pTgIx-ucMsY?t=14095), so that was my only thought. It also feels slightly invalid omit an event handler now that `isOpen` is fully controlled

Reply:
sounds good to me! thanks for the reference